crates/evm/src/types.rs

```diff
@@ -0,0 +1,51 @@
+//! Includes the types used by the citrea-evm.
+use std::collections::BTreeMap;
+
+use reth_primitives::{Address, B256};
+use serde::{Deserialize, Serialize};
+
+/// BlockOverrides is a set of header fields to override.
+#[derive(Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(default, rename_all = "camelCase", deny_unknown_fields)]
+pub struct BlockOverrides {
```

Comment:
https://reth.rs/docs/reth/rpc/types/struct.BlockOverrides.html

Reply:
That exists. However, our `number` and `time` vars are u64 but theirs are U256. If i wanted to change those in our code to U256, i would have to change a LOT of places.

Reply:
Unfortunately our block.number must be u256 IIRC

Reply:
Ended up using the type defined in Reth as you referred and use `saturating_to` to convert to u64 as @eyusufatik suggested.